src/casp/cromwell/bq_ops/extract/inputs_extract.json

```diff
@@ -0,0 +1,8 @@
+{
+  "CASExtractBQ.project_id": "dsp-cell-annotation-service",
```

Comment:
See comment above about using `CASExtractBQ.extract. ... ` if you want to not have to pass inputs from the workflow to the task